src/all/batoto/src/eu/kanade/tachiyomi/extension/all/batoto/BatoTo.kt

```diff
@@ -309,23 +322,36 @@ open class BatoTo(
         }
         return super.mangaDetailsRequest(manga)
     }
+    private var titleRegex: Regex =
+        Regex("(?:\\([^()]*\\)|\\{[^{}]*\\}|\\[(?:(?!]).)*]|«[^»]*»|〘[^〙]*〙|「[^」]*」|『[^』]*』|≪[^≫]*≫|﹛[^﹜]*﹜|𖤍.+?𖤍|/.+?)\\s*|([|/~].*)", RegexOption.IGNORE_CASE)
```

Comment:
this seems to be matching entire titles sometimes?
![image](https://github.com/user-attachments/assets/83df971c-b6d1-4d2a-8478-09e0a58ec2a4)
https://regex101.com/r/cViPD2/1

Reply:
It's strange but I've been using this regex since June and it never did anything like that...

Reply:
I can merge it then, someone will eventually open an issue if it causes problems

Reply:
This regex leaves a trailing space after the title is striped. Better also remove it.

Reply:
> This regex leaves a trailing space after the title is striped. Better also remove it. Try this: `\s*(?:\([^()]*\)|\{[^{}]*\}|\[(?:(?!]).)*]|«[^»]*»|〘[^〙]*〙|「[^」]*」|『[^』]*』|≪[^≫]*≫|﹛[^﹜]*﹜|𖤍.+?𖤍|/.+?)\s*|\s*([|/~].*)` Or add a trim() after replace regex.

Ty
I've added .trim() after replace regex